AK/FlyString.cpp

```diff
@@ -178,4 +178,13 @@ ErrorOr<void> Formatter<FlyString>::format(FormatBuilder& builder, FlyString con
     return Formatter<StringView>::format(builder, fly_string.bytes_as_string_view());
 }
 
+bool FlyString::equals_ignoring_ascii_case(FlyString const& other) const
+{
+    if (*this == other)
+        return true;
+    // FIXME: Rename StringUtils::equals_ignoring_case to equals_ignoring_ascii_case.
+    // FIXME: Make sure this function behaves correctly in the presence of non-ASCII characters.
```

Comment:
I noticed that during your video: it does. It only converts chars that happen to be upper case ASCII and leaves everything else alone. Non-ASCII code units are left alone.

(The function `to_ascii_lower_case` technically requires a code point, not a code unit, but due to the design of UTF-8 it just works out.)

Reply:
Ah right, I recalled there was some neat thing like that about UTF-8 but didn't feel entirely confident in my memory. Will drop the FIXME, thank you!